content-src/components/Spotlight/Spotlight.js

```diff
@@ -3,9 +3,11 @@ const React = require("react");
 const SpotlightItem = React.createClass({
   render() {
     const site = this.props;
+    const imageUrl = site.images[0].url;
```

Comment:
Same as Nick's comment above, do we need to check if `sites.images[0]` is non-null before trying to access the `url` property here?